### DISCUSSION TITLE
OTBR add-on e HA core

### DISCUSSION THREAD

**Discussion by frabar55** [indomus/forum#3232] [HUB: Home Assistant]
salve, il mio problema è semplice: come fare ad utilizzare OTBR add-on in una installazione HA core (docker). So che gli add-on funzionano solo con HAOS o supervised, ma non c'è modo di "integrarlo" anche in HA core?
grazie

**Comment by KC-inDomus** [comment]
https://indomus.it/guide/installare-e-configurare-openthread-border-router-con-docker-su-linux-debian-mini-pc-intel-nuc-o-altro/

**Reply by frabar55** [reply]
grazie, ho seguito il tutorial ma otbr non è partito. Non è partita la interfaccia web. riporto di seguito alcune righe del log. Domanda: xchè eth0 se il codice della ethernet è enpis0?
grazie

Default: mDNSResponder (Engineering Build) (Sep  6 2024 15:08:58) starting
Sep 21 16:49:33 f17821628888 otbr-web[172]: [INFO]-WEB-----: Running 0.3.0-d4fa5d3
Sep 21 16:49:33 f17821628888 otbr-web[172]: [INFO]-WEB-----: Border router web started on wpan0
Sep 21 16:49:33 f17821628888 otbr-web[172]: [ERR ]-WEB-----: OpenThread daemon is not running.
Sep 21 16:49:33 f17821628888 otbr-agent: [NOTE]-AGENT---: Backbone interface: eth0
Sep 21 16:49:33 f17821628888 otbr-agent[199]: [NOTE]-AGENT---: Running 0.3.0-d4fa5d3
Sep 21 16:49:33 f17821628888 otbr-agent[199]: [NOTE]-AGENT---: Thread version: 1.3.0
Sep 21 16:49:33 f17821628888 otbr-agent[199]: [NOTE]-AGENT---: Thread interface: wpan0
Sep 21 16:49:33 f17821628888 otbr-agent[199]: [NOTE]-AGENT---: Radio URL: spinel+hdlc+uart:///dev/ttyUSB0?uart-baudrate=115200
Sep 21 16:49:33 f17821628888 otbr-agent[199]: [NOTE]-ILS-----: Infra link selected: eth0
Sep 21 16:49:33 f17821628888 otbr-agent[199]: [INFO]-RCP_HOS-: OpenThread log level changed to 5
Sep 21 16:49:33 f17821628888 otbr-agent[199]: 49d.18:32:55.844 [D] P-SpinelDrive-: Sent spinel frame, flg:0x2, iid:0, tid:0, cmd:RESET
Sep 21 16:49:33 f17821628888 otbr-agent[199]: 49d.18:32:55.844 [D] P-SpinelDrive-: Waiting response: key=0

**Reply by frabar55** [reply]
salve, sono sempre fermo sullo stesso punto, per quante ne faccia, non riesco a tirare su la rete Thread, non c'è verso!
nessuno può darmi qualche dritta? Grazie in anticipo
